xml/depl_nodes.xml

```diff
@@ -198,18 +195,25 @@
   </para>
   <para>
     &desig; uses pool(s) over which it can distribute zones and
-    records, these pool(s) are not created by &crow;. Pools can have
-    varied configuration. Any misconfiguration can lead to
+    records. Pools can have varied configuration. Any misconfiguration can lead to
     information leakage.
   </para>
   <para>
-    One possible configuration of pools is created by &crow; on a node
-    with <literal>designate-api</literal> role in
-    <filename>/etc/designate/pools.crowbar.yaml</filename>. Copy
-    this file and edit it according to the requirements. Then provide this
+    &desig; barclamp creates default Bind9 pool out of the box, which can be
```

Comment:
The desig entity is lower case, so I suggest...`
The &desig; barclamp`...

Reply:
Done, in next commit.